bundles/mapping/toolbar/default-buttons.js

```diff
@@ -229,13 +229,14 @@ Oskari.clazz.category(
             const linkParams = sandbox.generateMapLinkParameters({});
             const viewUuid = this._getLinkUuid();
 
+            const layerSwipeActive = sandbox?.findRegisteredModuleInstance('LayerSwipe')?.active;
```

Comment:
And this is outdated as well after #2550. We should implement the getStateParameters() function for layerswipe instead as it is now stateful. This way we don't need to handle it separately on the link creation.